config/rbac/role.yaml

```diff
@@ -6,6 +6,26 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
I think both resources lack the 'get' permission :thinking:

Reply:
The current code does not need it, but given that it already needs perms like "watch" and "list" maybe it makes sense to add "get" too

Reply:
hmm that's interesting... what service account are you using for the pod? maybe by default get permission are added automatically to some common K8s objects...

Reply:
The current code performs Get on those resources:
https://github.com/3scale-labs/limitador-operator/blob/c9f1fa864c4d5fc812868df98b3061128f9202a0/controllers/limitador_controller.go#L94

so if it's working somehow it has those permissions

Reply:
I would add `get` too. It is not a big deal and you save future potential issues.

Reply:
Added `get` :+1:

Reply:
Yeah I would add it too :+1: . Looks good to me